File: PassPast/PassPast/Data/DataModels/Exam.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace PassPast.Data.DataModels
{
    public class Exam : BaseDataModel
    {
        public int Year { get; set; }
		public string Semester { get; set; }
        public Paper Paper { get; set; }
        public string Type { get; set; }
        public ICollection<Question>Questions { get; set; }
		public User CreatedBy { get; set; }
	}
}

File: PassPast 98/PassPast.Web/App/src/courses/models/exam.ts
import { Paper } from './paper';
import { Question } from './question';

export interface Exam{
    id: number;
    year: number;
    semester: string;
    paperId: number;
    paper: Paper;
    questions: Question[];
    createdById: number;
    createdBy: string;
}

